academic_observatory_workflows/unpaywall_telescope/unpaywall_telescope.py

```diff
@@ -608,12 +608,14 @@ def transform(release: dict, **context):
                     "transform: find and replace the 'authenticated-orcid' string in the jsonl to 'authenticated_orcid'"
                 )
                 for changefile in release.changefiles:
-                    find_replace_file(
-                        src=changefile.extract_file_path,
-                        dst=changefile.transform_file_path,
-                        pattern="authenticated-orcid",
-                        replacement="authenticated_orcid",
-                    )
+                    with open(changefile.extract_file_path, "r") as file_in:
```

Comment:
This might look nicer with both open() statements in the same context manger

```
with open(file1, "r") as f_in, open(file2, "w") as f_out:
```
Up to you though

Reply:
Thanks, I'll change that